blocks/linkable-cards/linkable-cards.css

```diff
@@ -37,44 +62,43 @@
     border-radius: 50%;
 }
 
-.linkable-cards.neutral-bone-style>div::after {
+.linkable-cards.neutral-bone-style > div::after {
   background: url("../../icons/download-arrow-purple.svg") no-repeat center;
   border: 2px solid var(--primary-purple);
 }
 
-.linkable-cards>div:hover {
+.linkable-cards > div:hover {
   transition: all .2s ease-out;
   box-shadow: 2px 4px 4px 2px rgb(0 0 0 / 15%);
 }
 
-.section .linkable-cards>div:hover::after {
+.section .linkable-cards > div:hover::after {
   cursor: pointer;
   transition: all .2s ease-out;
-  background: url("../../icons/download-arrow-white.svg") no-repeat center, linear-gradient(270deg,#fa1e32,#90f); 
+  background: url("../../icons/download-arrow-white.svg") no-repeat center, linear-gradient(270deg,#fa1e32,#90f);
```

Comment:
You could leverage `--gradient-left-right` here.

Reply:
Yes, good find. We are catching more and more of these to convert every time.